source/analysis/AM1DCalibrationAB.h

```diff
@@ -40,7 +40,7 @@ class AM1DCalibrationAB : public AMStandardAnalysisBlock
 	Q_OBJECT
 
 	Q_PROPERTY(QString dataName READ dataName WRITE setDataName)
-	Q_PROPERTY(QString NormalizationName READ NormalizationName WRITE setNormalizationName)
+	Q_PROPERTY(QString NormalizationName READ normalizationName WRITE setNormalizationName)
```

Comment:
No, but when I check to make sure they didn't, I learned something odd - possibly helpful?

See #2256